govuk/sign-in/utils/url/index.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const {URL} = require('url');
+
+function build(
+    base,
+    urlParts = {
+        search: {}
+    }
+) {
+    const builtUrl = new URL(base);
+    if (Object.keys(urlParts.search).length) {
+        builtUrl.search = new URLSearchParams(urlParts.search).toString();
+    }
+    return builtUrl.toString();
+}
+
+module.exports = {
+    build
+};
```

Comment:
I think there's value in having a centralised function that deals with the various URLs CW needs to know about to interact with the govuk service, but the actual building of the URL itself was done in one line before, albeit with a hard coded suffix. I'm not sure this is an improvement.